src/libraries/System.Numerics.Vectors/tests/Generation/Vector2.template

```diff
@@ -0,0 +1,1169 @@
+[Fact]
```

Comment:
These files need a license header too, which might impact how the templates are structured slightly.